pkg/pool-cow/coverage.sh

```diff
@@ -0,0 +1,124 @@
+#!/bin/bash
+
+set -e # exit on error
+
+# Functions
```

Comment:
Is this a soft-link?

Reply:
No. We're not using soft-links because the CI was breaking since https://github.com/balancer/balancer-v3-monorepo/pull/836. We could try to use them again, and check that CI stopped breaking.